module_3/gift_certificates_system/model/src/main/java/com/epam/esm/util/UrlMapping.java

```diff
@@ -0,0 +1,41 @@
+package com.epam.esm.util;
+
+/**
+ * Url mapping for controllers.
+ */
+public final class UrlMapping {
+
+    /**
+     * General URLs.
+     */
+    public static final String ID = "/{id}";
+
+    /**
+     * URL for Gift certificate entities
+     */
+    public static final String GIFT_CERTIFICATES = "/gift-certificates";
+
+    /**
+     * URL for Tag
+     */
+    public static final String TAGS = "/tags";
+    public static final String MOST_WIDELY_USED_TAG_OF_USER_WITH_HIGHEST_COST_OF_ALL_ORDERS = "/most-widely";
```

Comment:
From my point of view, path could be more descriptive

Reply:
Please, give an example.